applications/prompt-proto-service-lsstcomcamsim/values-usdfprod-prompt-processing.yaml

```diff
@@ -10,30 +10,33 @@ prompt-proto-service:
   image:
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: 2.5.0
+    tag: 4.1.0
 
   instrument:
     pipelines:
       main: >-
-        (survey="ops-rehearsal-3")=[${PROMPT_PROCESSING_DIR}/pipelines/LSSTComCamSim/ApPipe-noForced.yaml,
+        (survey="BLOCK-297")=[${PROMPT_PROCESSING_DIR}/pipelines/LSSTComCamSim/ApPipe.yaml,
         ${PROMPT_PROCESSING_DIR}/pipelines/LSSTComCamSim/SingleFrame.yaml,
         ${PROMPT_PROCESSING_DIR}/pipelines/LSSTComCamSim/Isr.yaml]
         (survey="")=[]
       preprocessing: >-
-        (survey="ops-rehearsal-3")=[]
+        (survey="BLOCK-297")=[]
         (survey="")=[]
     calibRepo: s3://rubin-summit-users
 
   s3:
     imageBucket: rubin-summit
-    endpointUrl: https://s3dfrgw.slac.stanford.edu
+    endpointUrl: https://sdfembs3.sdf.slac.stanford.edu
 
   imageNotifications:
     kafkaClusterAddress: prompt-processing-2-kafka-bootstrap.kafka:9092
-    topic: rubin-prompt-processing-prod
+    topic: rubin-summit-notification
 
   apdb:
-    config: s3://rubin-summit-users/apdb_config/sql/pp_apdb_lsstcomcamsim.py
+    config: s3://rubin-summit-users/apdb_config/cassandra/pp_apdb_lsstcomcamsim_pre_or4.py
+
+  alerts:
+    topic: alerts-simulated
```

Comment:
Just checking that this should not also be `alert-stream-test` as in dev.

Reply:
That's my understanding from @bsmartradio  from  https://lsstc.slack.com/archives/C0345PUU4N9/p1718147016397699